components/sections/Gallery.jsx

```diff
@@ -15,6 +16,7 @@ export default function Gallery() {
           );
         })}
       </div>
+      <img src="/assets/imgs/default.png" alt="" className="def hidden" />
```

Comment:
Can we use the Image component from Next here

Reply:
In the canvas tag when you draw with the Image component you will get an empty design.

Reply:
I guess if it was even possible it would have some lazy loading during the image appearance